crates/red_knot_python_semantic/src/semantic_index.rs

```diff
@@ -47,8 +48,10 @@ pub(crate) fn semantic_index(db: &dyn Db, file: File) -> SemanticIndex<'_> {
 /// is unchanged.
 #[salsa::tracked]
 pub(crate) fn symbol_table<'db>(db: &'db dyn Db, scope: ScopeId<'db>) -> Arc<SymbolTable> {
-    let _span = tracing::trace_span!("symbol_table", ?scope).entered();
-    let index = semantic_index(db, scope.file(db));
+    let file = scope.file(db);
+    let _span =
+        tracing::trace_span!("symbol_table", scope=?scope.as_id(), file=?file.path(db)).entered();
```

Comment:
Snuck in some tracing improvements, too! Is this related to the Salsa change? Either way, I like it.

Reply:
It's because `debug` now prints the entire struct and not just the id. This was very verbose, so I had to truncate it.